src/shapes/text.class.js

```diff
@@ -1320,32 +1345,40 @@
         lineLeftOffset = this._getLineLeftOffset(i);
         boxStart = 0;
         boxWidth = 0;
-        lastDecoration = this.getValueOfPropertyAt(i, 0, type);
-        lastFill = this.getValueOfPropertyAt(i, 0, 'fill');
+        decoration = this.getValueOfPropertyAt(i, 0, type);
+        fill = this.getValueOfPropertyAt(i, 0, 'fill');
+        top = topOffset + maxHeight * (1 - this._fontSizeFraction);
+        size = this.getHeightOfChar(i, 0);
+        dy = this.getValueOfPropertyAt(i, 0, 'deltaY');
         for (var j = 0, jlen = line.length; j < jlen; j++) {
           charBox = this.__charBounds[i][j];
-          currentDecoration = this.getValueOfPropertyAt(i, j, type);
-          currentFill = this.getValueOfPropertyAt(i, j, 'fill');
-          if ((currentDecoration !== lastDecoration || currentFill !== lastFill) && boxWidth > 0) {
-            ctx.fillStyle = lastFill;
-            lastDecoration && lastFill && ctx.fillRect(
+          _decoration = this.getValueOfPropertyAt(i, j, type);
+          _fill = this.getValueOfPropertyAt(i, j, 'fill');
+          _size = this.getHeightOfChar(i, j);
+          _dy = this.getValueOfPropertyAt(i, j, 'deltaY');
+          if ((_decoration !== decoration || _fill !== fill || _size !== size || _dy !== dy) && boxWidth > 0) {
```

Comment:
had to shorten variables' names to bring this line below 120 chars

Reply:
just break the line, is valid for eslint. you have to leave the boolean operator a the end of the line.

npm run lint

will give you results.

Reply:
ok